src/output.rs

```diff
@@ -27,11 +27,16 @@ impl OutputType {
     /// Try to launch the pager. Fall back to stdout in case of errors.
     fn try_pager(quit_if_one_screen: bool, pager_from_config: Option<&str>) -> Result<Self> {
         let pager_from_env = env::var("BAT_PAGER").or_else(|_| env::var("PAGER"));
-        let pager = pager_from_config
+
+        let mut pager = pager_from_config
             .map(|p| p.to_string())
             .or(pager_from_env.ok())
             .unwrap_or(String::from("less"));
 
+        if pager == "bat" {
```

Comment:
I think a better place to check for this would be a few lines below. See how we check for `less` in the `let is_less = ..` line. This also has the benefit of detecting things like `/usr/bin/bat`.

Reply:
Done!